documentation/modules/proc-managing-user-permissions-cli.adoc

```diff
@@ -14,13 +14,23 @@ You can edit the permissions for an existing user using the command line.
 
 .Procedure
 
-. In a text editor, open the desired user file, make the desired permissions changes, and save the file.
+. Retrieve the user whose permissions you want to edit:
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} get messaginguser user1 -o yaml > user1-example.yaml
```

Comment:
Sorry, this should probably be myspace.user1, as the user name is prefixed with the address space name.

Reply:
Done